settings.jsonc.example

```diff
@@ -0,0 +1,26 @@
+{
+    // Leave empty to print to stdout.
+    "LogFilePath": "ddns.log",
+    // The interval for periodic checks in seconds, 0 for an one-shot run.
+    "Interval": 300,
+    // Just need DNS edit permission, create at https://dash.cloudflare.com/profile/api-tokens
+    "OAuthToken": "<OAuthToken>",
+    // You can find ZONE_ID and RECORD_ID in Chrome Devtools.
+    "IPv6": {
+        "Enable": true,
+        "Proxied": "false",
+        "Hostname": "example.com",
+        "CloudflareUri": "https://api.cloudflare.com/client/v4/zones/<ZONE_ID>/dns_records/<RECORD_ID>"
+    },
+    "IPv4": {
+        "Enable": true,
+        "Proxied": "false",
+        "Hostname": "example.com",
+        "CloudflareUri": "https://api.cloudflare.com/client/v4/zones/<ZONE_ID>/dns_records/<RECORD_ID>",
+        "AddressAPI": "https://api.ipify.org/"
+    },
+    // Resolve the hostname and compare with current IP to decide whether to send an update request.
+    "EnableCompare": true,
```

Comment:
`Enable` -> `Enabled`

Reply:
Sorry, GitHub seems to have messed up the position. `EnableCompare` is fine. My comment was targeting `IPv4.Enable` and `IPv6.Enable`.